profiles/common/modules/custom/tmgmt_poetry/tmgmt_poetry.info

```diff
@@ -1,5 +1,6 @@
 name = Poetry Translator
 description = TMGMT implementation of Poetry European Service.
+php = 5.4
 package = Translation Management
 core = 7.x
 dependencies[] = date_popup
```

Comment:
This is not in the scope of this ticket.
For more information, you can follow the link:
https://github.com/ec-europa/platform-dev/commit/fb738fff403365511a0a8c6b7f78e757974a6a4d